guides/Makefile

```diff
@@ -1,37 +1,29 @@
 SHELL := /bin/bash
-ALL_DIRS = $(shell ls -d doc-*)
-GUIDES-HTML = $(ALL_DIRS)
+SUBDIRS = $(shell ls -d doc-*)
 
-.PHONY: all html pdf linkchecker all-html all-pdf clean all-clean linkchecker-tryer $(GUIDES-HTML)
+.PHONY: all clean html pdf linkchecker linkchecker-tryer serve subdirs $(SUBDIRS)
 
 all: html
 
-html: all-html
+html: subdirs
 
-pdf: all-pdf
+pdf: subdirs
 
-clean: all-clean
+subdirs: $(SUBDIRS)
+
+$(SUBDIRS):
+	$(MAKE) -C "$@" $(MAKECMDGOALS)
+
+clean:
+	@for DIR in $(SUBDIRS) ; do \
+		$(MAKE) -s --directory="$$DIR" clean ; \
+	done
```

Comment:
This is completely simplified, it now properly calls make so recursive option does work.